packages/valory/skills/abstract_round_abci/base.py

```diff
@@ -1215,8 +1213,6 @@ class CollectionRound(AbstractRound, ABC):
 
     # allow_rejoin is used to allow agents not currently active to deliver a payload
     _allow_rejoin_payloads: bool = False
-    # if the payload is serialized to bytes, we verify that the length specified matches
-    _hash_length: Optional[int] = None
```

Comment:
This was at the wrong level of abstraction. Moved to the `transaction_settlement_abci`